test/java/seedu/addressbook/commands/SortCommandTest.java

```diff
@@ -0,0 +1,45 @@
+package seedu.addressbook.commands;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import seedu.addressbook.data.AddressBook;
+import seedu.addressbook.data.person.*;
+import seedu.addressbook.util.TestUtil;
+
+import java.util.Collections;
+import java.util.List;
+
+public class SortCommandTest {
+
+

```

Comment:
Stray line break? Try to avoid these spaces by checking diff while submitting a PR.